dev/CommonStyles/Common_themeresources_any.xaml

```diff
@@ -109,6 +91,8 @@
             <Color x:Key="SolidBackgroundFillColorTertiary">#282828</Color>
             <Color x:Key="SolidBackgroundFillColorQuarternary">#2C2C2C</Color>
 
+            <Color x:Key="SystemFillColorAttention">#50C0FF</Color>
```

Comment:
This color resource should be deleted from all three themes.

Reply:
@marksfoster already removed :)